Makefile

```diff
@@ -40,12 +40,12 @@ test: checkall testall
 
 testfast:
 	@echo 'TESTSUITE'
-	cargo test --features=with_test_support
+	cd sentry && cargo test --features=with_test_support
 .PHONY: testfast
 
 testall:
 	@echo 'TESTSUITE'
-	cargo test --all-features --all
+	cargo test --all-features
```

Comment:
Does this activate all features of all workspace members? This was intended to test all targets.